### PR TITLE
Add docs for all 3 use cases of ChatQnA examples and change models for switch case

### PR DESCRIPTION
**Pull request by zhlsunshine**
## Description

Change list:

1. Add all 3 use cases of ChatQnA examples
2. Add reference link in GMC README
3. change the switch case models to `openlm-research/open_llama_3b`

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

`n/a`.

## Tests

This change contains doc and yaml file which can trigger automatically e2e test.

### CODE REVIEW

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+   ```sh
+   kubectl get service -n chatqa
+   ```
+
+3. Retrieve the application access URL
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n chatqa
+   NAME     URL                                                      READY     AGE
+   chatqa   http://router-service.chatqa.svc.cluster.local:8080      8/0/8     3m
```
Comment (KfreeZ) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:64]: can you confirm it's the latest controller? the latest controller should set the status to 9/0/9 as expected, including the the router's status

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+   ```sh
+   kubectl get service -n chatqa
+   ```
+
+3. Retrieve the application access URL
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n chatqa
+   NAME     URL                                                      READY     AGE
+   chatqa   http://router-service.chatqa.svc.cluster.local:8080      9/0/9     3m
```
Comment (KfreeZ) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:138]: for DataPrep, it's 10/0/10

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+   ```sh
+   kubectl get service -n switch
+   ```
+
+3. Retrieve the application access URL
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n switch
+   NAME     URL                                                   READY     AGE
+   switch   http://router-service.switch.svc.cluster.local:8080   14/0/14   83s
```
Comment (KfreeZ) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:191]: 15/0/15

File: microservices-connector/config/samples/ChatQnA/use_cases.md (hunk 1 above)
Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:64]: Hi @KfreeZ, this use case does not support the `dataprep` microservice.  `9/0/9` should be in `ChatQnA with data preparation`

Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:64]: Okay, got it, the number is increased

File: microservices-connector/config/samples/ChatQnA/use_cases.md (hunk 3 above)
Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:191]: got it

File: microservices-connector/config/samples/ChatQnA/use_cases.md (hunk 2 above)
Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:138]: got it

**Review by KfreeZ — APPROVED** [review]

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n chatqa
+   NAME     URL                                                      READY     AGE
+   chatqa   http://router-service.chatqa.svc.cluster.local:8080      10/0/10    3m
+   ```
+
+> [NOTE]
+
+Comparing with `General ChatQnA with preset RAG data`, there should be `9` microservices, the extra one is the microservice of `dataprep`.
```
Comment (yongfengdu) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:143]: Here should be 10 microservices? The above output is 10/0/10 but here mentioned 9, it's a little confusing if that's not wrong.

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+   ```sh
+   kubectl get gmconnectors.gmc.opea.io -n switch
+   NAME     URL                                                   READY     AGE
+   switch   http://router-service.switch.svc.cluster.local:8080   15/0/15   83s
+   ```
+
+> [NOTE]
+
+Comparing with `General ChatQnA with preset RAG data`, there should be `14` microservices, the extra are the microservices for different embedding models and LLM models.
```
Comment (yongfengdu) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:196]: Ditto

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+8. Check that the tgi-svc-deployment has been changed to use the new LLM Model
+
+   ```sh
+   kubectl get deployment tgi-service-m-deployment -n chatqa -o jsonpath="{.spec.template.spec.containers[*].env[?(@.name=='LLM_MODEL_ID')].value}"
+   ```
+
+9. Access the updated pipeline using the same URL from above using the client pod
+
+   ```sh
+   kubectl exec "$CLIENT_POD" -n chatqa -- curl -s --no-buffer $accessUrl -X POST -d '{"text":"What is the revenue of Nike in 2023?","parameters":{"max_new_tokens":17, "do_sample": true}}' -H 'Content-Type: application/json'
```
Comment (yongfengdu) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:109]: Please double check the question. I'm not sure if the nke.pdf data is imported or not by default. If there is no nike data import, you should change a question here.

Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:109]: Hi @yongfengdu, okay, I change the question as the same below.

File: microservices-connector/config/samples/ChatQnA/use_cases.md (hunk 4 above)
Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:143]: Nice catch! Done!

File: microservices-connector/config/samples/ChatQnA/use_cases.md (hunk 5 above)
Comment (zhlsunshine) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:196]: Thanks, done!

**Review by yongfengdu — APPROVED** [review]

**Pull request merged**